library/X509/Job.php

```diff
@@ -103,6 +109,11 @@ private static function generateTargets(ConfigObject $jobDescription, array $hos
             $start = static::addrToNumber($start_ip);
             for ($i = 0; $i < $ip_count; $i++) {
                 $ip = static::numberToAddr(gmp_add($start, $i), $ipv6);
+                if (self::isExcluded($excludes, $ip)) {
+                    Logger::info("Excluding IP '$ip' from scan target");
```

Comment:
Remove quotes around IP and remove the word target. I also would set the level to debug.